performance_test_factory/src/cli_options.cpp

```diff
@@ -73,8 +75,9 @@ void Options::parse(int argc, char ** argv)
       std::to_string(
         resources_sampling_per_ms)), "msec")(
     "x, executor",
-    "the system executor:\n\t\t\t\t1:SingleThreadedExecutor. 2:StaticSingleThreadedExecutor",
-    cxxopts::value<int>(executor)->default_value(std::to_string(executor)), "<1/2>")(
+    "system executor:\n\t\t\t\t1:SingleThreadedExecutor. 2:StaticSingleThreadedExecutor. \
+    3:EventsExecutor.",
+    cxxopts::value<int>(executor)->default_value(std::to_string(executor)), "<1/2/3>")(
```

Comment:
why the `std::to_string` ? the value seems to be an int

Reply:
It is an `int`, but according to what I was able to see, the library expects a `const string&`. GCC error below:
```
/root/floorcare_ws/src/ros2-performance/performance_test_factory/src/cli_options.cpp:79:50: error: cannot convert ‘int’ to ‘const string&’ {aka ‘const std::__cxx11::basic_string<char>&’}
   79 |     cxxopts::value<int>(executor)->default_value(executor), "<1/2/3/4>")(
      |                                                  ^~~~~~~~
      |                                                  |
      |                                                  int

```

Also, based on the examples from the rest of the arguments in the surrounding code, looks like `int` always get converted to `std::string` with this approach. Do you think there might be something to improve here?

Reply:
Ok, that's fine.
I don't really remember how this API works but we don't need to change it if it's not trivial